src/sentry/api/bases/organization_events.py

```diff
@@ -28,11 +28,13 @@ def get_orderby(self, request):
         if orderby:
             return orderby
 
-    def reference_event(self, request, organization):
+    def reference_event(self, request, organization, params):
```

Comment:
Why have a generic `params` in the signature? Why not explicit `start` and `end` arguments?